activesupport/lib/active_support/log_subscriber.rb

```diff
@@ -139,8 +139,12 @@ def logger
       LogSubscriber.logger
     end
 
+    def logger_level
+      logger.level.is_a?(Symbol) ? ::Logger.const_get(logger.level.upcase) : logger.level
```

Comment:
Hum, my worry here is that this is quite expensive for something that may be a hotspot.

If we really want to do this, I'd use a constant hash instead:

```ruby
LEVELS = {
  info: Logger::INFO,
  ...
}.freeze

def logger_level
  level = logger.level
  LEVELS.fetch(level, level)
end

```

Reply:
Implemented, thank you!